.storybook/pages/Cad/Cad.tsx

```diff
@@ -16,208 +16,591 @@ import {
+  },
+  {
+    title: 'Planned Projects',
+  },
+];
+
+const StandardsRows = [
+  {
+    value1: 'NGSS.3-5-ETS1-1',
+    projects: [
+      {
+        ariaLabel: 'Project 1',
+        complete: true,
+      },
+      {
+        ariaLabel: 'Project 2',
+        complete: true,
+      },
+      {
+        ariaLabel: 'Project 3',
+        complete: false,
+      },
+      {
+        ariaLabel: 'Project 4',
+        complete: false,
+      },
+      {
+        ariaLabel: 'Project 5',
+        complete: false,
+      },
+      {
+        ariaLabel: 'Project 6',
+        complete: false,
+      },
+      {
+        ariaLabel: 'Project 7',
+        complete: false,
+      },
+      {
+        ariaLabel: 'Project 8',
+        complete: false,
+      },
+      {
+        ariaLabel: 'Project 9',
+        complete: false,
+      },
+      {
+        ariaLabel: 'Project 10',
+        complete: false,
+      },
+    ],
+  },
+  {
+    value1: 'NGSS.3-5.ESS1.C',
+    projects: [
+      {
+        ariaLabel: 'Project 1',
+        complete: true,
+      },
+    ],
+  },
+  {
+    value1: 'NGSS.3-5.ESS1.E',
+    projects: [
+      {
+        ariaLabel: 'Project 1',
+        complete: true,
+      },
+      {
+        ariaLabel: 'Project 2',
+        complete: true,
+      },
+      {
+        ariaLabel: 'Project 3',
+        complete: true,
+      },
+      {
+        ariaLabel: 'Project 4',
+        complete: true,
+      },
+      {
+        ariaLabel: 'Project 5',
+        complete: true,
+      },
+    ],
+  },
+  {
+    value1: 'NGSS.3-5.PS3.C',
+    projects: [
+      {
+        ariaLabel: 'Project 1',
+        complete: false,
+      },
+      {
+        ariaLabel: 'Project 2',
+        complete: false,
+      },
+      {
+        ariaLabel: 'Project 3',
+        complete: false,
+      },
+      {
+        ariaLabel: 'Project 4',
+        complete: false,
+      },
+      {
+        ariaLabel: 'Project 5',
+        complete: false,
+      },
+    ],
+  },
+  {
+    value1: 'NGSS.3-5.PS3.E',
+    projects: [
+      {
+        ariaLabel: 'Project 1',
+        complete: false,
+      },
+      {
+        ariaLabel: 'Project 2',
+        complete: true,
+      },
+      {
+        ariaLabel: 'Project 3',
+        complete: false,
+      },
+      {
+        ariaLabel: 'Project 4',
+        complete: false,
+      },
+      {
+        ariaLabel: 'Project 5',
+        complete: false,
+      },
+    ],
+  },
+];
+
+export const Cad = () => {
+  const [indexState, setIndexState] = useState<number | undefined>(2);
+  const [containers, setContainers] = useState({
+    'container-1': {
+      itemIds: ['item-1', 'item-2', 'item-3', 'item-4', 'item-5'],
+      emptyContent: (
+        <>
+          <Text as="div" className="u-margin-bottom-xl">
+            <p>
+              There are no more available projects to create your course plan
+            </p>
+          </Text>
+          <img alt="hand with pencil" src={EmptyImage} />
+        </>
```

Comment:
Could you maybe pull all of these jsx blocks out into variables above? I feel like that might make it easier to track the logic here and reason about the state changes. Can be in a follow-up PR.